test/07-openqa-label-known-issues.t

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash -xv
+
+source test/init
+
+plan tests 7
+source openqa-label-known-issues
+client_args=(api --host "$host_url")
+
+export KEEP_REPORT_FILE=1
+
+client_output=''
+mock-client-output() {
+    client_output+="client_call $@"$'\n'
+    echo "$client_output\n"
+}
+
+client_call=(mock-openqa-cli "${client_call[@]}")
+logfile1=$dir/test/data/04-autoinst.txt
+
+try-client-output() {
+  #out=$logfile1
+  client_output=''
+  try "$*"' && echo "$client_output"'
+}
+openqa-cli() {
+    local id=$(basename "$4")
+    cat "$dir/test/data/$id.json"
+}
+curl() {
+    warn "URL $7"
+    if [[ "https://openqa.opensuse.org/tests/404/file/autoinst-log.txt" = "$7" ]]; then
+	echo "404"
+    fi
+    echo "200"
```

Comment:
If you're debugging variables, it can really help to put quotes around it to see where it starts and endes. Otherwise it can easily be overlooked that `$curl_output` has the wrong content here.

Reply:
Also there is no new line at the end when calling `curl -w "%{http_code}"` so you likely want to use `echo -n 200` to mimic that.